### PR TITLE
Translating to swedish

### PR DESCRIPTION
**Pull request by HannesAlexandersson**
I have added swedish versions of message.json in a sv folder in locales. 
I ahve added myself to the contributers list. 
I have added a sv folder in assets/texts and translated the files. The changelogs I only translated the 2 latest. 

Thanks for letting me contribute, this was my first ever time!

### CODE REVIEW

File: assets/texts/sv/Changelog/1.1.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* **Featured:** Save QR code size manually or automatically.
```
Comment (rugk) [inline assets/texts/sv/Changelog/1.1.md:1]: Note you don't need to copy non-translated files, you can just ignore/delete them. But it's fine, if you want remove them though… :slightly_smiling_face:

File: src/_locales/sv/messages.json [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+    "message": "Aktivera debug läge",
+    "description": "This is an option shown in the add-on settings."
+  },
+  "optionDebugModeDescr": {
+    "message": "Detta är endast användbart för att få mer detaljerad loggning i konsolen för att rapportera buggar, osv.",
+    "description": "The description shown for the debug mode (optionDebugMode)."
+  },
+
+  "translatorCredit": {
+    "message": "Detta tillägg har översatts till svenska av Hannes Alexandersson (@HannesAlexandersson) .",
```
Comment (rugk) [inline src/_locales/sv/messages.json:366]: Basically please keep the placeholder (`$…$`) here in order to have a real link to click on.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

Actually, it is possible [if you could try out the development version](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#coding) you can see how that all works and if it works for you.

Just so we know the translation works good in the browser when it is actually used.

**Review by rugk — CHANGES_REQUESTED** [review]

File: src/_locales/sv/messages.json (hunk 2 above)
Comment (HannesAlexandersson) [reply src/_locales/sv/messages.json:366]: Ive changed the placeholder link according to your request.
Ive tested it in development version now and it seems good. The swedish translation works fine in the context.

File: assets/texts/sv/Changelog/1.1.md (hunk 1 above)
Comment (HannesAlexandersson) [reply assets/texts/sv/Changelog/1.1.md:1]: Ok, I went ahead and just translated them all, i figured I might aswell do that.

**Review by HannesAlexandersson — COMMENTED** [review]
I have adjusted the message.json so it now correctly uses the placeholder links
Skärmbild 2024-09-04 164823

I also translated the changelogs instead of deleting the ones that where not translated before.

**Comment by HannesAlexandersson** [comment]
Hello and thanks for the feedback. This is actually my very first time
contributing to open source and Im not sure to what to do now? Ive
completed the requested changes in a codespace at github and committed the
changes aswell as "reviewed" them. I think I have correctly submitted the
changes but please let me know if I didnt do it correctly.
Best of days to you // Hannes

Den ons 4 sep. 2024 kl 13:10 skrev rugk ***@***.***>:

**Review by rugk — APPROVED** [review]
Thanks a lot! Yeah you did everything correctly, better than many others, possibly hehe. 

This will be released with the next update. :slightly_smiling_face:

**Pull request merged**